tests/pipeline/test_domain.py

```diff
@@ -314,15 +338,22 @@ def _test_equity_calendar_domain(self,
 
     def test_equity_calendar_domain(self):
         # test the defaults
-        self._test_equity_calendar_domain(US_EQUITIES, datetime.time(8, 45))
+        self._test_equity_calendar_domain(BE_EQUITIES, datetime.time(8, 15))
```

Comment:
It might be worth moving this block into its own test named something like `test_built_in_equity_calendar_domain_defaults`. It's useful to have coverage of the default definition for all the built-in domains, but the original purpose of this test was to be a test for the `EquityCalendarDomain` class itself, not for each built-in instance, so this is kind of testing two different things at this point.